src/xwayland/xutil.h

```diff
@@ -0,0 +1,52 @@
+#ifndef _WLC_XATOMS_H_
+#define _WLC_XATOMS_H_
+
+#include <stdint.h>
+#include <xcb/xcb.h>
+
+enum atom_name {
+   WL_SURFACE_ID,
+   WM_DELETE_WINDOW,
+   WM_TAKE_FOCUS,
+   WM_PROTOCOLS,
+   WM_NORMAL_HINTS,
+   MOTIF_WM_HINTS,
+   TEXT,
+   UTF8_STRING,
+   CLIPBOARD,
+   CLIPBOARD_MANAGER,
+   TARGETS,
+   PRIMARY,
+   WM_S0,
+   WLC_SELECTION,
+   NET_WM_S0,
+   NET_WM_PID,
+   NET_WM_NAME,
+   NET_WM_STATE,
+   NET_WM_STATE_FULLSCREEN,
+   NET_WM_STATE_MODAL,
+   NET_WM_STATE_ABOVE,
+   NET_SUPPORTED,
+   NET_SUPPORTING_WM_CHECK,
+   NET_WM_WINDOW_TYPE,
+   NET_WM_WINDOW_TYPE_DESKTOP,
+   NET_WM_WINDOW_TYPE_DOCK,
+   NET_WM_WINDOW_TYPE_TOOLBAR,
+   NET_WM_WINDOW_TYPE_MENU,
+   NET_WM_WINDOW_TYPE_UTILITY,
+   NET_WM_WINDOW_TYPE_SPLASH,
+   NET_WM_WINDOW_TYPE_DIALOG,
+   NET_WM_WINDOW_TYPE_DROPDOWN_MENU,
+   NET_WM_WINDOW_TYPE_POPUP_MENU,
+   NET_WM_WINDOW_TYPE_TOOLTIP,
+   NET_WM_WINDOW_TYPE_NOTIFICATION,
+   NET_WM_WINDOW_TYPE_COMBO,
+   NET_WM_WINDOW_TYPE_DND,
+   NET_WM_WINDOW_TYPE_NORMAL,
+   ATOM_LAST
+};
```

Comment:
I'm afraid not, it used to be in xwm.c i believe. I just moved it to a header since selection.c and xwm.c need it